k8s/aat/common/bsp/bulk-scan-sample-app.yaml

```diff
@@ -21,6 +21,30 @@ spec:
       replicas: 2
       useInterpodAntiAffinity: true
       image: hmctspublic.azurecr.io/bulk-scan/sample-app:prod-3b895c9b
+      testsConfig:
+        memoryLimits: "3072Mi"
```

Comment:
may be this much memory is not needed ?

Reply:
removed,
we do not set devmode so will use default .